### DISCUSSION TITLE
Seeing a huge increase in client load time between v2.23 and 2.24

### DISCUSSION THREAD

**Discussion by jwld** [prisma/prisma#8461] [Q&A] [answered]
Tested this with a file where I'm just importing the client:
```
const { PrismaClient } = require('@prisma/client')
```

With 2.23 this loads in under a second, whereas on 2.24 and beyond it takes roughly 12-13 seconds each time.

This is a bit of a major issue for us when it comes to our testing. Is this a known issue, and is there any way to reduce the load time?

**Comment by janpio** [comment, marked as answer]
This is not a known issue and has not been reported before. If it was widespread, I am very sure this would have popped up a lot of times, so this probably has something to do with your specific setup or project.

Do you have a reproduction? Then you can definitely open a bug issue and we will take a look.

**Reply by jwld** [reply]
Thanks will move over to the issue I opened https://github.com/prisma/prisma/issues/8484 and add more information tomorrow

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
